Add Redisinsight v1.12.1 release notes. (#2021)
RedisLabs/redislabs-docs: content/ri/release-notes/v1.12.0.md

weight: 2
---

## 1.12.1 (July 2022)

This is the maintenance release of RedisInsight 1.12 (v1.12.1)!

### Critical Bug Fix:
- Core:

>   - When adding or removing a Redis Enterprise Software or Redis Enterprise Cloud database in RedsInsight v1 that has the RediSearch module loaded, all hashes within that database would be deleted.

Reviewer: ```suggestion
  - When you add or remove a Redis Enterprise Software or Redis Enterprise Cloud database in RedsInsight v1 that has the RediSearch module loaded, all hashes within that database are deleted.
```

### Fixes:
- Core:
  - Added curl command to container.
  - Fixed container vulnerabilities (CVE-2022-1292, CVE-2022-2068).
- RediSearch:

>   - Fixed bug where index info doesn't report right information for RediSearch v2.

Reviewer: ```suggestion
  - Fixed index info to report correct information for RediSearch v2.
```

- Profiling:

>   - Added support for profiling MODULE commands.

Reviewer: ```suggestion
  - Added support for profiling module commands.
```

>   - Added support for viewing information of clients using ipv6 addresses.

Reviewer: ```suggestion
  - Added support for viewing information of clients that use IPv6 addresses.
```

- RedisGraph:

>   - Added support for `RO_QUERY` only mode. Now RedisGraph can work with just `RO_QUERY` command.

Reviewer: ```suggestion
  - Added support for `RO_QUERY` only mode. RedisGraph now responds to the `RO_QUERY` command.
```
